### PR TITLE
Hyperswitch: Removed HyperSwitch code from RESTBase

### PR DESCRIPTION
**Pull request by Pchelolo**
Hehe, day of huge PRs. Removed `HyperSwitch` from `RESTBase` codebase!

### CODE REVIEW

**Comment by d00rman** [comment]
I'm sooo tempted to hit the merge button :)))))

**Comment by d00rman** [comment]
Ups, @Pchelolo needs a rebase already. That's gonna be fun!

**Comment by Pchelolo** [comment]
@d00rman heh.. We should stop touching `/lib` files in RESTBase repo until this is merged :)

File: sys/key_rev_large_value.js [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-var mwUtil = require('../lib/mwUtil');
-var HTTPError = require('../lib/exports').HTTPError;
-var URI = require('swagger-router').URI;
 var preq = require('preq');
 
-var yaml = require('js-yaml');
-var fs = require('fs');
-var spec = yaml.safeLoad(fs.readFileSync(__dirname + '/key_rev_value.yaml'));
+var mwUtil = require('../lib/mwUtil');
+var HyperSwitch = require('hyperswitch');
```
Comment (d00rman) [inline sys/key_rev_large_value.js]: Minor nit: I'd prefer to first have the require list of modules, and then requires of local files. Mixing them kind of decreases readability.

Comment (d00rman) [inline sys/key_rev_large_value.js]: @Pchelolo ping?

Comment (Pchelolo) [inline sys/key_rev_large_value.js]: Oh, sorry, I've missed that comment. 5 mins

**Comment by d00rman** [comment]
I think we need to resolve and merge this ASAP, otherwise it may become impossible to keep track of the changes. Some nits in-lined though

**Comment by gwicke** [comment]
:+1: 

Lets merge this once Travis has +1ed as well, and unless @eevans has any objections.

**Pull request merged**